database/WiredTiger

```diff
@@ -0,0 +1,2 @@
+WiredTiger
+WiredTiger 11.2.0: (November 10, 2022)
```

Comment:
אנילא חושבת שהתיקיה הזאת צריכה לעלות לגיטהאב

Reply:
אין לי שמץ מה זה התקיה הזו - לא פגשתי אותה מעודי

Reply:
תמחקי אותה ותוסיפי לגיטיגנור

Reply:
OK
אבל יש שם עוד מלא קבצים מוזרים מקוה שהם לא יעשו את אותה השגיאה
C:\fullStack1\Project\communication\database

Reply:
משהו מוזר מחקתי והוא כותב לי nothing to commit